code/__DEFINES/dcs/signals/signals_datum.dm

```diff
@@ -31,3 +31,17 @@
 #define COMSIG_MERGER_REMOVING "comsig_merger_removing"
 /// Called on the merger after finishing a refresh: (list/leaving_members, list/joining_members)
 #define COMSIG_MERGER_REFRESH_COMPLETE "comsig_merger_refresh_complete"
+
+// Gas mixture signals
+/// Called when a gasmix have air added into it.
+#define COMSIG_GASMIX_MERGING "comsig_gasmix_merged"
+/// Called when a gasmix have air being subtracted from it.
+#define COMSIG_GASMIX_REMOVING "comsig_gasmix_removing"
+/// Called when a gasmix is currently reacting.
+#define COMSIG_GASMIX_REACTING "comsig_gasmix_reacting"
+
+// Modular computer's file signals. Used to tell the program datum something is going on.
+/// You are being added to a modular computer's HDD
+#define COMSIG_MODULAR_COMPUTER_FILE_ADDED "comsig_modular_computer_file_added"
+/// You are being deleted in a modular computer's HDD
+#define COMSIG_MODULAR_COMPUTER_FILE_DELETED "comsig_modular_computer_file_deleted"
```

Comment:
Are these events? or procs

Reply:
It tells a computer file that it's being deleted so other things can react to it. Event i suppose?
https://github.com/vincentiusvin/tgstation/blob/97557595b44239b556f88e456cca1dcd64ac4638/code/modules/research/ordnance/scipaper_publisher.dm#L32